addons/gforces/functions/fnc_pfhUpdateGForces.sqf

```diff
@@ -64,16 +49,19 @@ GVAR(oldVel) = _newVel;
 */
 
 _average = 0;
-if (count GVAR(GForces) > 0) then {
+if !(GVAR(GForces) isEqualTo []) then {
     _sum = 0;
-    {
+    _sumDif = {
         _sum = _sum + _x;
-    } forEach GVAR(GForces);
-    _average = _sum / (count GVAR(GForces));
+        true
+    } count GVAR(GForces);
+    _average = _sum / _sumDif;
 };
 
-_classCoef = ACE_player getVariable ["ACE_GForceCoef",
-    getNumber (configFile >> "CfgVehicles" >> (typeOf ACE_player) >> "ACE_GForceCoef")];
+_classCoef = ACE_player getVariable "ACE_GForceCoef";
+if (isNil "_classCoef") then {
```

Comment:
I don't think this is cleaner as the previous code. As for performance increase; I expect we now got an if statement that is true more often then not. If anything, I'd expect this to be worse on average.